src/system.yml

```diff
@@ -212,15 +212,14 @@ packs:
     module: swnr
     name: one-roll-backing
     path: packs/one-roll-backing.db
-    system: swnr    
+    system: swnr
```

Comment:
looks like emacs caught some stray whitespace, this is unrelated. happy to remove if desired